tests/IntegrationTests/TokenAcquirerTests/TokenAcquirer.cs

```diff
@@ -33,6 +33,16 @@ public TokenAcquirer()
             TokenAcquirerFactory.ResetDefaultInstance(); // Test only
         }
 
+        [Fact]
+        public void TokenAcquirerFactoryDoesNotUseAspNetCoreHost()
```

Comment:
This test project only tests NET7.0 and NET8.0 it will never test netstandard, which is part of the change.
Would it make sense to add netstandard as part of the target to the test project?

Reply:
No longer needed with the fix

Reply:
Well we should still test the whole library for netstandard2.0 dont you think? It`s a release target after all.